regcore/tests/responses_tests.py

```diff
@@ -10,8 +10,9 @@ def test_user_error(self):
         response = user_error("my reason")
         self.assertEqual(400, response.status_code)
         self.assertEqual('application/json', response['Content-type'])
-        self.assertTrue('my reason' in response.content)
-        json.loads(response.content)   # valid json
+        response_text = response.content.decode('utf-8')
+        self.assertTrue('my reason' in response_text)
```

Comment:
Truth. I'll take a pass and swap those and related tests out for their 2.7 version